versioned/spi/src/main/java/org/projectnessie/versioned/VersionStore.java

```diff
@@ -99,8 +99,11 @@ Hash commit(
    * @throws ReferenceNotFoundException if {@code branch} or if any of the hashes from {@code
    *     sequenceToTransplant} is not present in the store.
    */
-  void transplant(
```

Comment:
Can you explain these changes a bit more. I dont see why they should be pushed down to the VersionStore

Reply:
existing VS functionality is no longer changed via this PR